### PR TITLE
Add project roadmap

### PR DESCRIPTION
**Pull request by petrasovaa**
This is the roadmap we discussed during the community meeting.
image

### CODE REVIEW

File: themes/grass/layouts/about/roadmap.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="section-sm bg-white mb-1">
+        <div class="row nm align-items-center">
+          <div class="col-2 text-center"><i class="fas fa-book fa-7x"></i></div>
+          <div class="col-10">
+            <h3 class="mt-2 mb-2">Documentation</h3>
+            <p>We are committed to offer complete and up to date documentation
+              for newcomers, users and contributors. Our goal is to:
```
Comment (veroandreo) [inline themes/grass/layouts/about/roadmap.html:127]:
```suggestion
              for newcomers, users and contributors. Our goals are to:
```

**Review by veroandreo — APPROVED** [review]

File: themes/grass/layouts/about/roadmap.html [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+      <div class="section-sm bg-white mb-1">
+        <div class="row nm align-items-center">
+          <div class="col-2 text-center"><i class="fas fa-cloud-download-alt
+          fa-7x"></i></div>
+          <div class="col-10">
+            <h3 class="mt-2 mb-2">Software distribution</h3>
+            <p>In order to make GRASS run as smoothly as possible on all
+              platforms, we plan to:
+              <ul>
+                <li>Release and maintain a GRASS conda package.</li>
```
Comment (neteler) [inline themes/grass/layouts/about/roadmap.html:56]:
```suggestion
            <p>In order to make GRASS GIS run as smoothly as possible on all
              platforms, we plan to:
              <ul>
                <li>Release and maintain a GRASS GIS conda package.</li>
```

File: themes/grass/layouts/about/roadmap.html [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+    <div class="container">
+      <div class="section-sm bg-white mb-1">
+        <div class="row nm align-items-center">
+          <div class="col-2 text-center"><i class="fas fa-rocket
+          fa-7x"></i></div>
+          <div class="col-10">
+            <h3 class="mt-2 mb-2">Software releases</h3>
+            <p>GRASS mission is to bring advanced geospatial technologies to the
+              world. To that end we plan to release one minor version per year and
+              micro versions as needed:
```
Comment (neteler) [inline themes/grass/layouts/about/roadmap.html:38]:
```suggestion
            <p>The GRASS GIS mission is to bring advanced geospatial technologies to
               the world. To that end we plan to release one minor version per year and
              micro versions as needed:
```

File: themes/grass/layouts/about/roadmap.html [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+      </div>
+      <div class="section-sm bg-white mb-1">
+        <div class="row nm align-items-center">
+          <div class="col-2 text-center"><i class="fas fa-seedling
+          fa-7x"></i></div>
+          <div class="col-10">
+            <h3 class="mt-2 mb-2">Sustainability</h3>
+            <p>Different people have volunteered their time and money to sustain
+              and support GRASS GIS over time. To ensure GRASS' long-term success, we
+              aim to:
```
Comment (wenzeslaus) [inline themes/grass/layouts/about/roadmap.html:168]: I have the following suggestions: different->diverse, volunteered->invested, money->financial support, sustain and support...over time->ensure the ongoing development

```suggestion
            <p>A diverse group of contributors have invested their time and financial
             support to ensure the ongoing development of GRASS GIS. To ensure GRASS' long-term success, we
             aim to:
```

File: themes/grass/layouts/about/roadmap.html [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+
+    <div class="container">
+      <div class="section-sm bg-white mb-1">
+        <div class="row nm align-items-center">
+          <div class="col-2 text-center"><i class="fas fa-rocket
+          fa-7x"></i></div>
+          <div class="col-10">
+            <h3 class="mt-2 mb-2">Software releases</h3>
+            <p>GRASS mission is to bring advanced geospatial technologies to the
+              world. To that end we plan to release one minor version per year and
```
Comment (wenzeslaus) [inline themes/grass/layouts/about/roadmap.html:37]:
```suggestion
              world. To that end, we plan to release one minor version per year and
```

File: themes/grass/layouts/about/roadmap.html [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+      </div>
+      <div class="section-sm bg-white mb-1">
+        <div class="row nm align-items-center">
+          <div class="col-2 text-center"><i class="fas fa-bullhorn
+          fa-7x"></i></div>
+          <div class="col-10">
+            <h3 class="mt-2 mb-2">Communication</h3>
+            <p>GRASS GIS is used in several different ways by different users and in
+              different contexts. To showcase GRASS versatility and innovation we
+              aim to:
```
Comment (wenzeslaus) [inline themes/grass/layouts/about/roadmap.html:187]:
```suggestion
              different contexts. To showcase GRASS versatility and innovation, we
              aim to:
```

**Review by wenzeslaus — APPROVED** [review]
One wording and two comma consistency suggestions.

**Review by wenzeslaus — APPROVED** [review]

**Comment by neteler** [comment]
https://grass.osgeo.org/about/roadmap/

(one only finds it when clicking on "About Us")

**Comment by veroandreo** [comment]
> https://grass.osgeo.org/about/roadmap/
> 
> (one only finds it when clicking on "About Us")

Right, we forgot the menu entry. I'll add that in a moment.

**Comment by wenzeslaus** [comment]
Also consider naming it just Roadmap.

**Pull request merged**